### ISSUE TITLE
include autophagosomes-clem-datasets?

### ISSUE THREAD

**Issue by martinschorb** [mobie/volume-em-segmentation#1]
Hi,

would it be sensible to include data from:

https://github.com/mobie/autophagosomes-clem-datasets

?
Has this been published?
There is some interesting EM segmentation that might be useful.

**Comment by martinschorb** [comment]
also, @constantinpape does this one https://github.com/mobie/lgn-em-project contain any useful data and segmentations that we could use for training/transfer learning to improve segmentation?

**Comment by constantinpape** [comment]
> https://github.com/mobie/autophagosomes-clem-datasets

No, has not been published. We could ask if the data can be used.

> https://github.com/mobie/lgn-em-project contain any useful data and segmentations that we could use for training/transfer learning to improve segmentation?

There are other EM connectomics projects that would be more suitable (but nothing that's currently directly in MoBIE).


In general: can you give some more context of what you want to do exactly? There is a lot of data out there that is not in MoBIE, and I am working on some transfer learning approaches; so I can maybe give some more references with more context :). (If you don't want to explain in the issue here feel free to write by EMBL chat.)

**Comment by martinschorb** [comment]
Hi,

I am trying to collect volume-EM data with existing segmentations in this MoBIE project. The idea is to have a central go-to point to collect data from. Then we could just test segmentation models against this data and potentially use it for transfer learning.

Next, I will include Julian's famous HeLa cell. Feel free to dump other things (other EM imaging types) as well.
